src/core/ctf.cpp

```diff
@@ -410,6 +420,13 @@ float CTF::Evaluate(float squared_spatial_frequency, float azimuth) {
     }
 }
 
+// Return the value of the CTF at the given squared spatial frequency and azimuth taken into account the sample thickness
+// Formulas according to "TEM bright field imaging of thick specimens: nodes in Thon ring patterns" by Tichelaar, et.al.
```

Comment:
While listed in Tichelaar et al. aren't they referencing McMullan et al 2015 for this equation?

The sinc is because the slab thickness is being considered a "pixel" in the Z-direction, an idea that predates the McMullan paper. But the cosine term is added why? I thought Tichelaar concluded the cosine term was incorrect? (Or at least inconsistent with their numerical integration)

"Since we determined the object thickness and because we could detect higher nodes, we were also able to discriminate between the analytical expressions already proposed by others for describing the Thon ring patterns. The “top-hat” expression by McMullan et al. [2] yields a profile that agrees with our experimental results. ***However, their “cosine” approximation, as well as the expression derived by DeRosier [1], do not.***"

Either way, okay with having it as is for now - just trying to understand your logic.

Reply:
I think the question of top-hat vs cosine comes from the question, how to describe the density of the sample in the beam direction. (i.e. is the density uniform or is the sample denser in the center). 

The cosine in the formula here just comes from the identity sin^2(x) = 0.2(1-cos(2x)), so the formula here does represent the top-hat profile.

Reply:
Ah, I think you're right - had to go into the supplement. The cosine profile they were referring to 
![image](https://user-images.githubusercontent.com/3077528/202546297-e02f831c-dbbc-46a0-a111-bf92237ab688.png)